examples/index.json

```diff
@@ -708,6 +708,20 @@
             }
         ]
     },
+    {
+        "title": "Interface to external code",
+        "column": 2,
```

Comment:
This isn't something we want to encourage users to do often. I think this should go on the last column, before [Testing](https://ballerina.io/learn/by-example/#testing) maybe.

Reply:
Updated in 1875fa9427232e83ad168698f5c0fd774ee40859.